### PR TITLE
docs: add issues/pulls templates

### LINKED ISSUE #368: Add PR and issue templates
Would help walk people through adding themselves to the contributors list (for a PR) and also help with titling.

### PR DESCRIPTION
**Pull request by andrewda**
Closes #368

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+## Actual vs. Expected
+
+<!--
+  Describe what happened. What did you expect to happen?
+-->
+
+
+<!-- DO NOT MODIFY BELOW THIS LINE -->
+--------------------------------------
+<!-- GITPOINT_BUG -->
```
Comment (andrewda) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md]: We can use this for auto tagging issues in the future if we make some kind of GitPoint bot.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  Bonjour!
+
+  We can't express how grateful we are that you're working on making GitPoint
+  better! We're thrilled to take a look at the changes you've made and merge
+  them in as soon as possible. Please fill out this template to make the
+  reviewal process as quick and smooth as possible!
+
+  Thanks again for your hard work!
+-->
+
```
Comment (machour) [inline .github/PULL_REQUEST_TEMPLATE.md]: We should add more questions/checklist here, to encourage/make sure that:

- Unit tests are provided
- The work was tested on a simulator or device (maybe ask for the OS version too)
- The contributor used `yarn contributors:add` to add him or herself as a contributor




- [ ] Tests provided
- [ ] I have tested this PR

**Comment by machour** [comment]
Super nice @andrewda! Left you some comments for things to enhance

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  https://gitter.im/git-point
+
+  Thanks!
+-->
+
+## Phone OS and Model
+
+<!--
+  Please specify the OS version and model of phones that this bug has been
+  tested on.
```
[the remark this answers is not in the document]
Comment (machour) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.md]: We should also ask for the GitPoint version number (displayed in user options)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (lex111) [reply .github/PULL_REQUEST_TEMPLATE.md]: For example, as is done in [Symfomy](https://github.com/symfony/symfony/blob/master/.github/PULL_REQUEST_TEMPLATE.md).

**Comment by andrewda** [comment]
@machour latest commits should have incorporated your suggestions!

**Pull request merged**